### DISCUSSION TITLE
Концепция лендинга "Онлайн курс"

### DISCUSSION THREAD

**Discussion by Darolga** [torin-asakura/drum-in#28] [General]
@TorinAsakura 

[Макет](https://www.figma.com/file/7FTyeashb3rns6sCdrsz3b/Landing?node-id=51%3A1424)

В блоке "Кому подойдет курс" текст с вопросами и ответами открывается при нажатии на стрелку, поверх всего блока. Закроется по крестику. [Референс](https://ulyanovsk.hh.ru/employer/4023)

![Online](https://user-images.githubusercontent.com/77831845/145991070-46e538aa-85b1-42ed-afc1-2047bb1accda.jpg)

![Popup](https://user-images.githubusercontent.com/77831845/145991095-3228294f-0245-4920-b4e3-1fe114386b80.png)

**Comment by MarinaDrumin** [comment]
круто

**Comment by Darolga** [comment]
@MarinaDrumin

Для сайта "Онлайн курс" ссылка на видео битая [https://disk.yandex.ru/d/aLqTs48OnZr9dQ](https://disk.yandex.ru/d/aLqTs48OnZr9dQ)

В разделе
Уровень 2. Пятое Измерение

FIVE MEASURE – видео-задания
Счёт 12345 доли и акценты
Чтение 5/8
Ключ 3-4
Чтение 9/8
Компинг на ф-но
Конокол 3
Фразы
Ключ 2-3 на ф-но
Конокол 4
https://disk.yandex.ru/d/aLqTs48OnZr9dQ
Свинг-соло
Конокол 6
Пирамидка с квинтолями

**Reply by MarinaDrumin** [reply]
https://youtu.be/h4TMh4XAOqI
ссылка первая

**Comment by Darolga** [comment]
@MarinaDrumin
Аналогично и здесь н работает ссылка на видео

Уровень 3. Седьмое Небо

SEVEN SKY - видео-задания
Счёт 1234567
Ключ 3-4 на ф-но
Конокол 5
https://disk.yandex.ru/d/kXPiJi4aykq0Gg

**Reply by MarinaDrumin** [reply]
поправлю в документе

**Reply by MarinaDrumin** [reply]
Вечером все поправлю
